### PR TITLE
Added custom player build support

### PR DESCRIPTION
**Pull request by moser4035**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitmovin-javascript",
-  "version": "1.9.0",
+  "version": "1.10.0-1-SNAPSHOT",
```
Comment (fabiandev) [inline package.json]: Should this be reverted to `1.9.0` before merging into develop?

File: examples/player/package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "bitmovin-javascript-examples",
+  "version": "0.0.1",
+  "dependencies": {
+    "bitmovin-javascript": "1.10.0-1-SNAPSHOT",
```
Comment (fabiandev) [inline examples/player/package.json]: Change to `~1.10.0` before merging, or will a snapshot version be released?

File: examples/player/01_custom_player_build.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    }).catch((error) => {
+      console.error('Error adding custom player build domain', error);
+      return reject(error);
+    });
+  });
+};
+
+const createCustomBuild = (playerVersion, domains) => {
+  return new Promise((resolve, reject) => {
+    const customBuild = {
```
Comment (flofischer) [inline examples/player/01_custom_player_build.js]: Not a question about the code but rather for general understanding: Does a custom build always consist of only the playerversion and the domains or are there any optional fields like name or description that were just not used in the example?

Comment (moser4035) [reply examples/player/01_custom_player_build.js]: There are the fields name and description but they are not used here

File: examples/player/package.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "bitmovin-javascript-examples",
+  "version": "0.0.1",
+  "dependencies": {
+    "bitmovin-javascript": "1.9.0",
```
Comment (fabiandev) [inline examples/player/package.json]: Shouldn't this be `^1.9.0` or `~1.10.0`?

File: examples/player/01_custom_player_build.js [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+const Promise = require('bluebird');
+
+const BITMOVIN_API_KEY = '<YOUR_API_KEY>';
+const bitmovin = new Bitmovin({apiKey: BITMOVIN_API_KEY, debug: false});
+
+const playerVersion = 'v7.6.2';
+const domains = ['example.com', 'bitmovin.com'];
+
+const main = () =>
+  new Promise((resolve, reject) => {
```
Comment (Domysee) [inline examples/player/01_custom_player_build.js]: Why create a new promise here and not just return the one from `Promise.all`?

File: examples/player/01_custom_player_build.js [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+            });
+        });
+      })
+      .catch(error => {
+        reject(error);
+      });
+  });
+
+const waitForPlayerBuildFinished = playerBuild => {
+  return new Promise((resolve, reject) => {
```
Comment (Domysee) [inline examples/player/01_custom_player_build.js]: Same here

File: examples/player/01_custom_player_build.js [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+          setTimeout(waitForPlayerBuildToBeFinishedOrError, 10000);
+        });
+    };
+    waitForPlayerBuildToBeFinishedOrError();
+  });
+};
+
+const createPlayerBuildDomain = domain => {
+  return new Promise((resolve, reject) => {
```
Comment (Domysee) [inline examples/player/01_custom_player_build.js]: Same here

File: examples/player/01_custom_player_build.js [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+      })
+      .catch(error => {
+        console.error('Error adding custom player build domain', error);
+        return reject(error);
+      });
+  });
+};
+
+const createCustomBuild = (playerVersion, domains) => {
+  return new Promise((resolve, reject) => {
```
Comment (Domysee) [inline examples/player/01_custom_player_build.js]: Same here

**Pull request merged**